mininet/node.py

```diff
@@ -656,7 +656,7 @@ class Docker ( Host ):
     """
 
     def __init__(
-            self, name, dimage, dcmd=None, **kwargs):
+            self, name, dimage, dcmd=None, rm=False, **kwargs):
```

Comment:
```suggestion
            self, name, dimage, dcmd=None, **kwargs):
```

Let us use the **kwargs dict to hold the `rm` variable. Would be cleaner.